src/sidebars/search/search-controller.js

```diff
@@ -19,15 +19,23 @@
             this._watch();
         }
 
+        hasStore() {
```

Comment:
I'm not 100% sure what you mean here, do you mean something like:
```js
var store = this.getStore();
if (store) { // something }
```
Instead of having to refer to `this.store` in the `if` block?